src/docformatter/configuration.py

```diff
@@ -102,6 +102,12 @@ def do_parse_arguments(self) -> None:
             action="store_true",
             help="only check and report incorrectly formatted files",
         )
+        self.parser.add_argument(
```

Comment:
Can you fix the linting error in this section, please?

Reply:
Sure, done!